pkg/options/options.go

```diff
@@ -130,6 +131,8 @@ func (s *KubeRouterConfig) AddFlags(fs *pflag.FlagSet) {
 		"Each node in the cluster will setup BGP peering with rest of the nodes.")
 	fs.BoolVar(&s.BGPGracefulRestart, "bgp-graceful-restart", false,
 		"Enables the BGP Graceful Restart capability so that routes are preserved on unexpected restarts")
+	fs.Uint16Var(&s.BGPGracefulRestartDeferralTime, "bgp-graceful-restart-deferral-time", 10,
```

Comment:
This should probably be a `DurationVar` and we should convert it to uint16 later?

Reply:
Or at least mention "in seconds" in the CLI help string